runtime/vm/CRIUHelpers.cpp

```diff
@@ -183,6 +183,12 @@ isDebugOnRestoreEnabled(J9JavaVM *vm)
 			&& isCRaCorCRIUSupportEnabled(vm);
 }
 
+BOOLEAN
+isTransitionToDebugInterpreterRequested(J9JavaVM *vm)
```

Comment:
I think this method can be inlined since it just checks one flag.

Reply:
I like keeping this as a function, but I will add the `VMINLINE` attribute.